src/components/organisms/Nav.js

```diff
@@ -132,14 +184,42 @@ const Nav = () => {
             confirmText="Sign Up"
             cancelHandler={() => setModalState(modalStates.closed)}
           >
+            <label htmlFor="name">
+              Display Name
+              <input
+                type="text"
+                name="name"
+                value={isUsername}
+                onChange={(e) => { handleNameChange(e); }}
+                onBlur={(e) => { validation(e); }}
```

Comment:
since all you're doing is passing in the event object, we don't need to write this out in anonymous functions. by just calling the name of your function, events will automatically give the function its event object by default

```suggestion
                onChange={handleNameChange}
                onBlur={validation}
```

Reply:
You resolved this but it hasn't been done.